challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz_test.go

```diff
@@ -0,0 +1,33 @@
+package solution
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestFindThreeLargest(t *testing.T) {
+	tables := []struct {
+		input []int
+		result []int
+	}{
+		{[]int{141, 2, 17, -7, -17, -27, 18, 541, 8, 7, 7}, []int{18, 141, 541}},
+		{[]int{11, -7, 5, -7}, []int{-7, 5, 11}},
+		{[]int{1}, nil},
+	}
+
+	for _, table := range(tables) {
```

Comment:
So, `Go::range()` is basically `Python::enumerate()`?

Reply:
yeah exactly like enumerate!